src/module.ts

```diff
@@ -74,6 +76,12 @@ export interface ModuleHooks {
   'scripts:registry': (registry: RegistryScripts) => void | Promise<void>
 }
 
+declare module '@nuxt/schema' {
```

Comment:
this shouldn't be needed if you've ran `pnpm dev:prepare`

Reply:
although it seems a bit flakey so maybe it's needed to avoid tscheck issues

Reply:
yes :+1: